packages/design-tokens/tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
+    "noEmit": false,
+    "strict": true,
+    "noImplicitAny": true,
+    "rootDir": ".",
+    "outDir": "build",
+    "declarationMap": true,
+    "declaration": true,
+    "target": "es5",
+    "module": "CommonJS"
```

Comment:
`module` is duplicated from the master tsconfig file, you can remove this line.

Reply:
I thought keeping this here would be safer because it's important to this package specifically, i.e. that design-tokens works on nodejs.
What do you think?